command/analyzer/dryrun/run.go

```diff
@@ -87,26 +91,46 @@ func (a *AnalyzerDryRun) AnalyzerRun() (err error) {
 		return err
 	}
 
-	// Building the Analyzer image.
-	a.Spinner.StartSpinnerWithLabel("Building Analyzer image...", "Built Analyzer image")
-	ctxCancelFunc, buildRespReader, buildError := a.Client.BuildAnalyzerDockerImage()
-
-	// Cancel the build context and close the reader before exiting this function.
-	if buildRespReader != nil {
-		defer buildRespReader.Close()
-	}
-	defer ctxCancelFunc()
-	if buildError != nil {
-		a.Spinner.StopSpinnerWithError("Failed to build the Analyzer image", err)
-		return buildError
+	err = a.Client.SetupClient()
+	if err != nil {
+		return err
 	}
 
-	// Check the docker build response.
-	if err = docker.CheckBuildResponse(buildRespReader, false); err != nil {
-		a.Spinner.StopSpinnerWithError("Failed to build the Analyzer image", err)
-		return err
+	if a.DockerImageName != "" {
+		a.Spinner.StartSpinnerWithLabel("Fetching Analyzer image...", "Fetched analyzer image")
```

Comment:
Mention the name of the image being pulled as well in the spinner message. Also, please replace `Fetching` with `Pulling`.